lib/ace/ext/papeeria/spellchecker.coffee

```diff
@@ -0,0 +1,16 @@
+define (require, exports, module) ->
+    "use strict";
+    # SpellChecker is a class that by now provides only numbers highlighting,
+    # but it will provide a hunspell spellchecking someday.
+    class SpellChecker
+        constructor: () ->
+
+        # Function checks whether token is numeric.
+        # @param {String} token Token to check.
+        # @return {Boolean} True if token is numeric, false otherwise.
+        check: (token) ->
```

Comment:
In CoffeeScript it is recommended to use `=>` rather than `->` in non-static methods - this allows for using `@` (`this` in CS world)